desktop/apps/show/components/follow_profile/index.coffee

```diff
@@ -1,18 +1,20 @@
+Profile = require '../../../../models/profile.coffee'
 CurrentUser = require '../../../../models/current_user.coffee'
 { Following, FollowButton } = require '../../../../components/follow_button/index.coffee'
 
-module.exports = (profile) ->
+module.exports = (profileId) ->
   user = CurrentUser.orNull()
   following = new Following [], kind: 'profile' if user
 
-  $el = $(".profile-follow[data-id='#{profile.id}']")
+  $el = $(".profile-follow[data-id='#{profileId}']")
+  profile = new Profile id: profileId
 
   new FollowButton
     el: $el
     following: following
     modelName: 'profile'
-    href: "#{profile.href}/follow"
```

Comment:
Not sure I understand. Are we passing in a function to the `href` option or `profile.href` is simply `undefined`? If latter, then wouldn't `profile.href()` fail because it attempts to call a function with `undefined`?

Reply:
`profile` was a Backbone model, so `profile.href` doesn't do anything, `profile.href()` is what is intended.

`profile.href` doesn't invoke the actual function:

<img width="266" alt="screen shot 2018-01-10 at 1 51 05 pm" src="https://user-images.githubusercontent.com/1457859/34789715-56891852-f60d-11e7-85c6-d5ff20dd1844.png">

Reply:
👍